lib/pdk/generate/module.rb

```diff
@@ -58,7 +58,8 @@ def self.invoke(opts = {})
 
         begin
           PDK::Module::TemplateDir.new(template_uri, metadata.data, true) do |templates|
-            templates.render do |file_path, file_content|
+            templates.render do |file_path, file_content, file_status|
```

Comment:
I must be missing something but where does `file_status` come from?

Reply:
Ah, I see, you linked that in the other thread: https://github.com/puppetlabs/pdk/blob/master/lib/pdk/module/templatedir.rb#L125-L130